### PR TITLE
update mega menu

### PR DESCRIPTION
**Pull request by samuel-523**

### CODE REVIEW

File: config/settings_data.json [hunk 1]
```diff
@@ -1,9 +1,9 @@
 {
   "current": {
-    "free_shipping_normal": "£150",
-    "free_shipping_pro": "£100",
-    "free_shipping_whs": "£50",
-    "store_country": "United States",
+    "free_shipping_normal": "$150",
+    "free_shipping_pro": "$100",
+    "free_shipping_whs": "$50",
+    "store_country": "Australia",
```
Comment (duchaiweb) [inline config/settings_data.json:6]: @samuel-523 this repo is US store, why did you chang this to Australia?

File: config/settings_data.json [hunk 2]
```diff
@@ -587,10 +579,10 @@
     "display_quickview_subtotal": true,
     "display_quickview_short_desc": true,
     "cart_notes": true,
-    "ajax_cart": true,
+    "ajax_cart": false,
```
Comment (duchaiweb) [inline config/settings_data.json:582]: @samuel-523 @abpalancajr i think we dont want to disable Ajax cart

File: config/settings_data.json [hunk 3]
```diff
@@ -587,10 +579,10 @@
...
     "show_shipping_calculator": true,
     "shipping_calculator_heading": "Get shipping estimates",
-    "shipping_calculator_default_country": "United Kingdom",
+    "shipping_calculator_default_country": "Australia",
```
Comment (duchaiweb) [inline config/settings_data.json:585]: @samuel-523 i'm not sure but this should be US

File: config/settings_data.json [hunk 4]
```diff
@@ -1000,7 +992,9 @@
     "border_color_product_item": "#ff0000",
     "product_item_border": "true",
     "product_main_height": "500",
-    "content_for_index": []
+    "content_for_index": [
+
```
Comment (duchaiweb) [inline config/settings_data.json:996]: @samuel-523 this array still is empty so we dont need to create new line for it

**Comment by duchaiweb** [comment]
@samuel-523 This pull request is done for what task? Can you paste the link of this task from Bugherd. So I think the branch name should have means. And you should separate the changes in config json files to another commit in order to revert easily if we need

**Comment by abpalancajr** [comment]
Cancelled!

**Pull request closed without merging**